### PR TITLE
Update to table format for alg parameters - GDAL raster projection

### PR DESCRIPTION
**Pull request by havatv**
Goal:

Ticket(s): #4429

- [x] Backport to LTR documentation is required

### CODE REVIEW

File: source/docs/user_manual/processing_algs/gdal/rasterprojections.rst [hunk 1]
```diff
@@ -42,139 +70,234 @@ This algorithm is derived from the `GDAL srsinfo utility <https://gdal.org/gdals
...
+   * - **Use multithreaded warping implementation**
+     - ``MULTITHREADING``
+     - [boolean]
+       
+       Default: False
+     - Two threads will be used to process chunks of the image and
+       perform input/output operations simultaneously.
+       Note that the computation itself is not multithreaded.
+   * - **Additional command-line parameters**
+       Source CRS
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/gdal/rasterprojections.rst]:
```suggestion

```

File: source/docs/user_manual/processing_algs/gdal/rasterprojections.rst [hunk 2]
```diff
@@ -79,102 +133,170 @@ This algorithm is derived from the `GDAL warp utility <https://gdal.org/gdalwarp
...
+       * 10 --- CFloat32
+       * 11 --- CFloat64
+       
+   * - **Georeferenced extents of output file to be created**
+       
+       Optional
+     - ``TARSource CRSGET_EXTENT``
+     - [extent]
+     - Sets the georeferenced extent of the output file to be
+       created (in the ``target CRS`` by default.
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/gdal/rasterprojections.rst]: Will fix (using guilabel).

File: source/docs/user_manual/processing_algs/gdal/rasterprojections.rst (hunk 1 above)
Comment (havatv) [reply source/docs/user_manual/processing_algs/gdal/rasterprojections.rst]: Already fixed.

File: source/docs/user_manual/processing_algs/gdal/rasterprojections.rst [hunk 3]
```diff
@@ -21,12 +21,40 @@ This algorithm is derived from the the `GDAL edit utility <https://gdal.org/gdal
...
+   :stub-columns: 0
+
+   * - Label
+     - Name
+     - Type
+     - Description
+   * - **Layer with projection**
+     - ``OUTPUT``
+     - [raster]
+     - The output raster layer (with a the projection information)
```
[the remark this answers is not in the document]
Comment (havatv) [reply source/docs/user_manual/processing_algs/gdal/rasterprojections.rst]: Already fixed.

**Comment by havatv** [comment]
I think and hope that all your suggestions will be reflected after I have merged two PRs (#4479 and #4480).

**Pull request merged**